docs/5.0/pages/cli-docs.mdx

```diff
@@ -41,7 +41,7 @@ the following services.
 | `--nodename` | `hostname` command on the machine | **string** | assigns an alternative name for the node which can be used by clients to login. By default it's equal to the value returned by
 | `-c, --config` | `/etc/teleport.yaml` | **string** `.yaml` filepath | starts services with config specified in the YAML file, overrides CLI flags if set
 | `--bootstrap` | none | **string** `.yaml` filepath | bootstrap configured YAML resources {/* TODO link how to configure this file */}
-| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
+| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node [env=dev,app=web]. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
```

Comment:
```suggestion
| `--labels` | none | **string** comma-separated list | assigns a set of labels to a node, for example [env=dev,app=web]. See the explanation of labeling mechanism in the [Labeling Nodes](admin-guide.mdx#labeling-nodes-and-applications) section.
```

Reply:
also, remove the square brackets
users might think they are needed in the flag value